debian/rules

```diff
@@ -1,9 +1,22 @@
 #!/usr/bin/make -f
 export DH_VERBOSE=1
+include /etc/os-release
```

Comment:
Nit picking, put this include next to the ifeq block, so it's clearer that the VERSION_ID variable comes from there

Reply:
done